velox/functions/lib/string/tests/StringImplTest.cpp

```diff
@@ -111,6 +111,103 @@ class StringImplTest : public testing::Test {
         {"\u1E8A", "\u1E8B"},
         {"\u1E8E", "\u1E8F"}};
   }
+
+  static std::vector<std::pair<std::string, std::string>>
+  getInitcapUnicodePrestoTestData() {
+    return {
```

Comment:
Hi @Joe-Abraham, there is an internal build error (listed below). I believe it is because we build Velox with C++20 internally, while `u8"..."` has the type `const char8_t[]` that's not directly convertible to std::string. Could you take a look?